wchat/screens/ContactList/components/ContactCell/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { AppRegistry, StyleSheet, Text, Image, View } from 'react-native';
+
+export default class ContactCell extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.content}>
+          <Image style={styles.avatar} source={{uri: this.props.avatar}} />
+          <View style={styles.info}>
+            <Text style={styles.name}>{this.props.username} </Text>
+            <Text style={styles.message}>Lorem ipsum dolor sit amet...</Text>
+          </View>
+        </View>
+        <View style={styles.separator} />
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
We extract all styles into a styles.js file at same level that index.js component, then import it to use styles.

Reply:
Like in react